src/node-red/echo-skill/customNodes/aas/i40-aas-onboarding.js

```diff
@@ -52,237 +60,238 @@ module.exports = function(RED) {
         frame: {
           semanticProtocol: 'i40:registry-semanticProtocol/onboarding',
           type: 'publishInstance',
-          messageId: new Date().getTime(),
+          messageId: '' + new Date().getTime(),
```

Comment:
here is the change... I do not know what is wrong with the formatter. I used 
`eclint fix src/node-red/echo-skill/customNodes/aas/i40-aas-onboarding.js`